.gitlab/tests.yml

```diff
@@ -85,13 +85,14 @@ build_base_venvs:
   retry: 2
   before_script:
     - !reference [.testrunner, before_script]
-    - pip install riot==0.20.1
+    - pip install riot==0.20.1 "setuptools_scm[toml]>=4"
     - unset DD_SERVICE
     - unset DD_ENV
     - unset DD_TAGS
     - unset DD_TRACE_REMOVE_INTEGRATION_SERVICE_NAMES_ENABLED
   script:
     - |
+      python -m setuptools_scm --force-write-version-files
```

Comment:
This line will create the `ddtrace/_version.py` file. Without it, importing `ddtrace` will attempt to fetch the version, won't find it, then will import `importlib_metadata` as a fallback mechinism.